tests/cancun/eip4844_blobs/test_blob_txs.py

```diff
@@ -305,22 +306,50 @@ def pre(  # noqa: D103
     """
     return {
         TestAddress: Account(balance=total_account_minimum_balance + account_balance_modifier),
+        TestAddress2: Account(balance=10**9),
```

Comment:
This seems to only be used by the blockchain test version of `test_invalid_tx_max_fee_per_blob_gas`, leading me to take a look at the usage of `non_zero_blob_gas_used_genesis_block` from `conftest.py`. Which also is only used in that test within `blob_txs.py`.

I think this threw me off when reviewing! My preference would be to keep the test how it was before, purely for readability. It also has added benefit of keeping the fixtures slightly smaller

 More below on the latter :)